packages/react-ui/components/Kebab/__stories__/Kebab.stories.tsx

```diff
@@ -130,6 +131,55 @@ WithFixedMenuHeight.story = { name: 'With fixed menu height', parameters: { cree
 export const KebabWithoutAnimations = () => <SomethingWithKebab disableAnimations size="small" />;
 KebabWithoutAnimations.story = { name: 'Kebab without animations', parameters: { creevey: { skip: [true] } } };
 
+export const ProgrammaticFocus: CSFStory = () => {
+  const kebab = React.useRef<Kebab | null>(null);
+
+  const focus = () => kebab.current?.focus();
+  const blur = () => kebab.current?.blur();
+
+  return (
+    <div style={{ width: 200, textAlign: 'center' }}>
+      <Gapped>
+        <button data-tid="focus-button" onClick={focus}>
+          Focus
+        </button>
+        <button data-tid="blur-button" onClick={blur}>
+          Blur
+        </button>
+        <Kebab size="small" ref={kebab} />
+      </Gapped>
+    </div>
+  );
+};
+
+ProgrammaticFocus.story = {
+  name: 'Kebab with programmatic focus',
+  parameters: {
+    creevey: {
+      tests: {
+        async callFocusThenBlur() {
+          const focusButton = await this.browser.findElement({ css: '[data-tid="focus-button"]' });
+          const blurButton = await this.browser.findElement({ css: '[data-tid="blur-button"]' });
+
+          await this.browser
+            .actions()
+            .click(focusButton)
+            .perform();
+          const afterFocus = await this.takeScreenshot();
+
+          await this.browser
+            .actions()
+            .click(blurButton)
+            .perform();
+          const afterBlur = await this.takeScreenshot();
+
+          await this.expect({ afterFocus, afterBlur }).to.matchImages();
```

Comment:
Давай немного переименуем, а то префикс `after` лишь мешает.
```suggestion
          await this.expect({ focus, blur }).to.matchImages();
```